src/main/java/org/folio/fqm/service/EntityTypeFlatteningService.java

```diff
@@ -90,17 +90,23 @@ private EntityType getFlattenedEntityType(UUID entityTypeId, EntityTypeSource so
             .stream()
             .filter(col -> !Boolean.TRUE.equals(source.getEssentialOnly()) || Boolean.TRUE.equals(col.getEssential()))
             .map(col -> col
-              // Don't use aliasPrefix here, since the prefix is already appropriately baked into the source aliass in flattenedSourceDefinition
```

Comment:
This comment seems useful enough to keep around, since it explains something that may not be obvious